modules/wildfly/src/main/modules/ogm/cassandra-drivers/module.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Hibernate OGM, Domain model persistence for NoSQL datastores
+ ~
+ ~ License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ ~ See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+  -->
+<module xmlns="urn:jboss:module:1.1" name="org.hibernate.ogm.cassandra.drivers" slot="${hibernate.ogm.cassandra.module.slot}">
```

Comment:
BTW while I think we should do it across all dialects, I've avoided that for now as it doesn't seem fit for a CR1->Final upgrade.. it requires some attention as for example you don't want to hide too much, for example to allow using native queries where supported.